.travis.yml

```diff
@@ -8,23 +8,31 @@ php:
   - 7.0
   - hhvm
 
-matrix:
-  allow_failures:
-    - php: 7.0
-    - php: hhvm
+sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - vendor
+
+matrix:
+  fast_finish: true
```

Comment:
Indeed my bad. If I'm right it should be added on a separate PR for discussion. Will do it later.